packages/shorebird_cli/test/src/commands/apps/create_apps_command_test.dart

```diff
@@ -34,7 +34,9 @@ void main() {
       codePushClient = _MockCodePushClient();
       command = CreateAppCommand(
         auth: auth,
-        buildCodePushClient: ({required String apiKey}) => codePushClient,
+        buildCodePushClient: ({required String apiKey, Uri? hostedUri}) {
```

Comment:
Do you just want a helper method in your test utils?

Reply:
I'm planning to refactor this in a subsequent PR to make it easier to write "middleware" so this should hopefully become simpler.